plugin/storage/grpc/options.go

```diff
@@ -28,16 +27,12 @@ import (
 )
 
 const (
-	pluginBinary             = "grpc-storage-plugin.binary"
-	pluginConfigurationFile  = "grpc-storage-plugin.configuration-file"
 	pluginLogLevel           = "grpc-storage-plugin.log-level"
```

Comment:
this looks like it's related to sidecar plugin model, not to remote grpc-storage